docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -2,43 +2,43 @@
 layout: nodes.liquid
 section: smartContract
 date: Last Modified
-title: "Consuming Price Feeds"
+title: "Consuming Price Feeds (EVM)"
 permalink: "docs/get-the-latest-price/"
 whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Price Feeds API Reference":"/docs/price-feeds-api-reference/", "Price Feeds Contract Addresses":"/docs/reference-contracts/"}
 metadata:
-  title: "Consuming Price Feeds"
+  title: "Consuming Price Feeds (EVM)"
   description: "How to use Chainlink Price Feeds in your smart contracts."
   image:
     0: "/files/OpenGraph_V3.png"
 ---
- {% markdown %}
+{% markdown %}
 
-Price data feeds are available on the following networks:
-- [EVM-compatible networks](#ethereum-network-examples)
-- [Solana network](#solana-network-examples)
+Data feeds are available on the following networks:
+- **EVM-compatible networks**
+- [Solana network](../get-the-latest-price-solana/)
 
-To get a full list of supported blockchains for price feeds, see the [Price Feeds Contract Addresses](../reference-contracts/) page.
+The list of price feeds for each network are available on the [Price Feed Contracts](../reference-contracts/) page.
+
+> 📘 New Feed Registry
+> You can use the [Feed Registry](../feed-registry/) to reference price feed assets by name or currency identifier instead of by pair/proxy address.
 
 > 📘 Note
 >
-> The full list of price feeds for each network are available from [Price Feed Contracts](../reference-contracts/).
-
-> 📘 New Feed Registry
-> You can now use the [Feed Registry](../feed-registry/) to reference price feed assets by name or currency identifier instead of pair/proxy address.
+> This guide shows you how to deploy and interact with price feeds using EVM networks. To learn how to use Solana networks, see [Solana networks](../get-the-latest-price-solana/).
 
 ## Ethereum network examples
```

Comment:
I know the example is specifically for Ethereum, but should we also say that the syntax etc can be used with any other EVM chain as well (eg BSC, Polygon)?

I guess it could be confusing because the title of the page is EVM Price Feeds, but then it just shows an Etheruem example. As a developer, would be great to know that I could use that example as well on any other EVM chains if  I come to this page

Reply:
Working on a solution here.

Reply:
Updated. How does the latest commit look?